lib/ansible/modules/cloud/misc/virt_clone.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Larry Smith Jr. <mrlesmithjr@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Requirements:
```

Comment:
Move this information to notes section, so user will be able to see using `ansible-doc` command. Just like [this](https://github.com/ansible/ansible/blob/0127a75132a6802526997496057235b893dcfcf1/lib/ansible/modules/cloud/vmware/vmware_guest.py#L31)